canvas_modules/common-canvas/src/common-properties/components/virtualized-table/virtualized-table.jsx

```diff
@@ -439,6 +444,7 @@ class VirtualizedTable extends React.Component {
 				className={classNames(className,
 					{ "properties-vt-row-selected": selectedRow },
 					{ "properties-vt-row-disabled": rowDisabled },
+					{ "properties-vt-row-doubleClick": this.getDoubleClickedRow()?.indexOf(rowData.rowKey) > -1 },
```

Comment:
Why do we need this?

Reply:
`properties-vt-row-doubleClick` class was added for styling the double clicked row(a blue border) which is not required for single click.